solutions/client-setup/folder.yaml

```diff
@@ -18,6 +18,6 @@ metadata:
   name: clients.client-name # kpt-set: clients.${client-name}
   namespace: hierarchy
 spec:
-  displayName: client-name # kpt-set: ${client-name}-${environment}
+  displayName: client-name # kpt-set: ${client-name}
```

Comment:
for consistency, if we remove the suffix, the environment is also hardcoded in 
 [experimentation/client-landing-zone](https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/e82ae774e40a3625e06bb649ee4091a4f547f7f3/solutions/experimentation/client-landing-zone/client-folder/folder.yaml#L21)

Reply:
implemented in latest commit